records_mover/records/types.py

```diff
@@ -18,150 +20,108 @@
 MutableRecordsHints = Dict[str, JsonValue]
 
 
-if TYPE_CHECKING:
-    from ..db import DBDriver  # noqa
-    # https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_EXTERNAL_TABLE.html
-    from mypy_extensions import TypedDict
+# https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_EXTERNAL_TABLE.html
+class RecordsManifestEntryMetadata(TypedDict):
+    content_length: int
+
 
-    class RecordsManifestEntryMetadata(TypedDict):
-        content_length: int
+class BaseRecordsManifestEntry(TypedDict):
+    url: str
+    mandatory: bool
 
-    class BaseRecordsManifestEntry(TypedDict):
-        url: str
-        mandatory: bool
 
-    class LegacyRecordsManifestEntry(BaseRecordsManifestEntry, total=False):
-        meta: RecordsManifestEntryMetadata
+class LegacyRecordsManifestEntry(BaseRecordsManifestEntry, total=False):
+    meta: RecordsManifestEntryMetadata
 
-    class RecordsManifestEntryWithLength(BaseRecordsManifestEntry):
-        meta: RecordsManifestEntryMetadata
 
-    class LegacyRecordsManifest(TypedDict):
-        entries: List[LegacyRecordsManifestEntry]
+class RecordsManifestEntryWithLength(BaseRecordsManifestEntry):
+    meta: RecordsManifestEntryMetadata
 
-    class RecordsManifestWithLength(TypedDict):
-        entries: List[RecordsManifestEntryWithLength]
 
-    class UrlDetailsEntry(TypedDict):
-        content_length: int
+class LegacyRecordsManifest(TypedDict):
+    entries: List[LegacyRecordsManifestEntry]
 
-    Url = str
-    UrlDetails = Dict[Url, UrlDetailsEntry]
 
-    from typing_extensions import Literal  # noqa
+class RecordsManifestWithLength(TypedDict):
+    entries: List[RecordsManifestEntryWithLength]
 
-    from mypy_extensions import TypedDict
 
-    #
-    # Note: Any expansion of these types should also be done in
-    # records.jobs.hints
-    #
-    RecordsFormatType = Literal['delimited', 'parquet']
+class UrlDetailsEntry(TypedDict):
+    content_length: int
 
-    DelimitedVariant = Literal['dumb', 'csv', 'bigquery', 'bluelabs', 'vertica']
 
-else:
-    RecordsManifestEntryMetadata = Mapping[str, int]
-    LegacyRecordsManifestEntry = Mapping[str, Union[str, bool, int, RecordsManifestEntryMetadata]]
-    RecordsManifestEntryWithLength =\
-        Mapping[str, Union[str, bool, int, RecordsManifestEntryMetadata]]
-    LegacyRecordsManifest = Mapping[str, List[LegacyRecordsManifestEntry]]
-    RecordsManifestWithLength = Mapping[str, List[RecordsManifestEntryWithLength]]
-    UrlDetailsEntry = Dict[str, int]
-    Url = str
-    UrlDetails = Dict[Url, UrlDetailsEntry]
+Url = str
+UrlDetails = Dict[Url, UrlDetailsEntry]
 
-    RecordsFormatType = str
 
-    DelimitedVariant = str
+#
+# Note: Any expansion of these types should also be done in
+# records.jobs.hints
+#
+RecordsFormatType = Literal['delimited', 'parquet']
 
-if TYPE_CHECKING:
-    HintEncoding = Literal["UTF8", "UTF16", "UTF16LE", "UTF16BE",
-                           "UTF16BOM", "UTF8BOM", "LATIN1", "CP1252"]
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker